shared/test/test_properties.rb

```diff
@@ -40,6 +46,17 @@ def test_get_set_delete
   # Methods below this line are test utilities, not actual tests
   private
 
+  def init_apis
+    @channels = Rack::Test::Session.new(Rack::MockSession.new(ChannelsApi, "studio.code.org"))
+
+    # Make sure the properties api has the same storage id cookie used by the channels api.
+    @channels.get '/v3/channels'
+    cookies = @channels.last_response.headers['Set-Cookie']
+    properties_mock_session = Rack::MockSession.new(PropertiesApi, "studio.code.org")
+    properties_mock_session.cookie_jar.merge(cookies)
+    @properties ||= Rack::Test::Session.new(properties_mock_session)
```

Comment:
What's the purpose of `||=` here?  I think `@properties` will always be unset when you get here.